code/script.js

```diff
@@ -200,15 +204,16 @@ const CHARACTERS = [
   },
 ]
 
-// Global variables
```

Comment:
The overall structure of the code is very neat and easy to follow from top to bottom.